build.ps1

```diff
@@ -7,20 +7,6 @@ $ErrorActionPreference = 'Stop'
 .\dev-env\windows\bin\dadew.ps1 sync
 .\dev-env\windows\bin\dadew.ps1 enable
 
-function bazel() {
-    Write-Output ">> bazel $args"
-    $global:lastexitcode = 0
-    $backupErrorActionPreference = $script:ErrorActionPreference
-    $script:ErrorActionPreference = "Continue"
-    & bazel.exe --bazelrc=.\nix\bazelrc @args 2>&1 | %{ "$_" }
-    $script:ErrorActionPreference = $backupErrorActionPreference
-    if ($global:lastexitcode -ne 0 -And $args[0] -ne "shutdown") {
```

Comment:
I think the exitcode logic here is still required.

Reply:
This logic is probably broken. I have seen it continue successfully when bazel.exe was missing on the PATH.

Restored for now as it's not essential to this PR.